### ISSUE TITLE
Loss getting too high

### ISSUE THREAD

**Issue by wellescastro** [juntang-zhuang/LadderNet#1]
Hi! 

I tried to execute your code and I've experienced a strange behavior regarding the training and validation losses. I didn't change any parameter, I've just cloned the repository and followed the instructions for the execution without change anything. These are the first four epochs:

```
Epoch 0: 
Train loss 0.347656
Valid loss: 5.1115

Epoch 1: 
Train loss 0.219913
Valid loss: 3.2819

Epoch 2: 
Train loss 0.175050
Valid loss: 3.2127

Epoch 3: 
Train loss 0.160487
Valid loss: 3.0469

Epoch 4: 
Train loss 1629511453423918336.000000
Valid loss: 18624358.1250

```
I've tried with the Pytorch 0.4.1 and Pytorch 0.4.0 and it didn't change the behavior. The loss keeps increasing as the training continues.

Extra info:
Python 3.6
CUDA 9.0
Ubuntu 14.04

**Comment by juntang-zhuang** [comment]
Hi,

Thanks for your interest. I found it was an error with the learning rate. I actually used an initial learning rate of 0.001, but I uploaded the wrong version of the code. 
Please see the updated version of retinaNN_training.py, where the learning rate is modified.

**Comment by wellescastro** [comment]
Thank you for the support, good work :)
